### ISSUE TITLE
Switch to feed-provided categories for tags to improve quality

### ISSUE THREAD

**Issue by ByteHamster** [gpodder/mygpo#376]
Hi,

I am currently trying to understand podcast tags. On https://gpodder.net/directory/, I see tags like `News & Politics=` or `"mac` (notice the special characters). If I then click on a podcast in those lists and view their tag list, the tags look a lot more "sane". Where do these tags come from and why are they different on different pages?

The tag lists on the podcast's pages contain something like `society & culture, gespräch, politik, news, anstöße, aktuelles`. If I use the tags API (https://gpodder.net/api/2/tags/100.json) to load existing tags, it most of the tags listed on the podcast page are not there. Why does the API only return some tags?

Could you please explain to me how the tags work? The current selection of tags that are returned from the API look a bit broken, to be honest.

**Comment by SiqingYu** [comment]
For now, the tags are under categories. They are organized in Django-admin
![image](https://user-images.githubusercontent.com/22290018/77133907-d2f77b00-6a9f-11ea-8d1e-b97c884d396b.png)
I think this is confusing too. Maybe we should remove tags altogether.

**Comment by ByteHamster** [comment]
Does this mean that tags are managed manually using Django-admin? Why do they have titles with special chars (`"`, `=`) all over them?

**Comment by stefankoegl** [comment]
Podcast tags have been retrieved from various sources. I assume that in some cases the extraction of the tags or the data provided by the sources was not correct, which caused those issues.

As the current data is not of high quality, it might be worth either to do thorough cleaning, or to even start from scratch and rely only on tags / categories from the feed.

**Comment by ByteHamster** [comment]
@stefankoegl Thanks!

> start from scratch and rely only on tags / categories from the feed

This sounds like the only sustainable way. Everything else needs manual work and will probably break again later. Also, it probably has good results because the feeds are usually tagged by humans.

@SiqingYu Let's make this issue a feature request to use the tags directly from the feeds. Could you please re-open? :)

**Comment by SiqingYu** [comment]
I agree with @stefankoegl. We should simplify this matter. I'm inclined to preserve category while removing tags. We can extract categories from websites like player.fm or Stitcher, which curate podcast categories by hand.
